compiler/rustc_ast_lowering/src/lib.rs

```diff
@@ -1466,7 +1466,7 @@ impl<'a, 'hir> LoweringContext<'a, 'hir> {
                             t.span,
                             E0562,
                             "`impl Trait` not allowed outside of {}",
-                            "function and inherent method return types",
+                            "function method return types",
```

Comment:
```suggestion
                            "function and method return types",
```

Reply:
This will likely affect some tests, so you need to re-bless

Reply:
Yep, everything is done. I was blessing tests locally :).